persist-cli-tests/src/test/java/io/ballerina/persist/tools/utils/GeneratedSourcesTestUtils.java

```diff
@@ -95,16 +95,18 @@ public static void assertGeneratedSources(String subDir, Command cmd) {
         }
     }
 
-    public static void assertGeneratedSourcesNegative(String subDir, Command cmd, String object) {
+    public static void assertGeneratedSourcesNegative(String subDir, Command cmd, String[] objects) {
```

Comment:
```suggestion
    public static void assertGeneratedSourcesNegative(String subDir, Command cmd, String[] relativeFilepaths) {
```

Reply:
As I understand, this is not objects, but file paths

Reply:
updated in [d1e9e41](https://github.com/ballerina-platform/persist-tools/pull/52/commits/d1e9e4195fff39c00f1fce312aa40487f7196262)